### PR TITLE
Nitesh home page

### PR DESCRIPTION
**Pull request by NManem**

### CODE REVIEW

File: client/App.jsx [hunk 1]
```diff
@@ -1,15 +1,23 @@
 import React from 'react';
 import { Routes, Route } from 'react-router-dom';
 import './styles/styles.scss';
-
+import ThemeContext from './components/ThemeContext';
```
Comment (derekoko) [inline client/App.jsx]: Update the import accordingly to pull in UserContext instead of ThemeContext

File: client/App.jsx [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
+
+  //& Using ThemeContext to have 'username' and 'setUserName' as global variables throughout are app (ITS OP!)
+  const [username, setUserName] = useState('currymonstanacho')
+
   return (
-    <Routes>
-      {/* <Route path="/" element={<Login />} /> */}
-      <Route path = '/home' element = {<Home/>}/>
-    </Routes>
+    <ThemeContext.Provider value={{username, setUserName}}>
```
Comment (derekoko) [inline client/App.jsx]: Change the context to be called UserContext.Provider to match proper naming convention

File: client/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React, {useContext} from 'react';
+import ThemeContext from './ThemeContext';
```
Comment (derekoko) [inline client/components/Header.jsx]: Change the context to be called UserContext and import accordingly

File: client/components/Tasks.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ThemeContext from './ThemeContext';
```
Comment (derekoko) [inline client/components/Tasks.jsx]: Change the context to be called UserContext. Also missing an import for { useContext} how are you using context on line 7 without that import?

File: client/components/ThemeContext.jsx [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { createContext } from 'react';
+
+//Initialize ThemeContext
+const ThemeContext = createContext();
```
Comment (derekoko) [inline client/components/ThemeContext.jsx]: Change naming convention to UserContext

File: client/components/ThemeContext.jsx [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+
+export default ThemeContext;
```
Comment (derekoko) [inline client/components/ThemeContext.jsx]: Don't export default. Please export UserContext and the import in other files should be import {UserContext} from '../path'

File: client/App.jsx [hunk 7]
```diff
@@ -1,15 +1,23 @@
...
+      <Routes>
+        {/* <Route path="/" element={<Login />} /> */}
+        <Route path='/home' element={<Home />} name={name} />
+      </Routes>
+    </ThemeContext.Provider>
```
Comment (derekoko) [inline client/App.jsx]: UserContext.Provider

**Review by derekoko — CHANGES_REQUESTED** [review]
Please fix changes before we can pull and merge.

**Review by AdrianAdamZ — APPROVED** [review]

**Pull request closed without merging**